core/services/ocr2/plugins/ocr2keeper/evmregistry/v21/logprovider/buffer_v2.go

```diff
@@ -0,0 +1,324 @@
+package logprovider
```

Comment:
Could this file live in a v2 package? 🙏 

So like, `core/services/ocr2/plugins/ocr2keeper/evmregistry/v21/logprovider/v2/buffer.go`

Reply:
Created AUTO-9256